src/awkward/_connect/rdataframe/from_rdataframe.py

```diff
@@ -71,7 +71,7 @@ def form_dtype(form):
     def empty_buffers(cpp_buffers_self, names_nbytes):
         buffers = {}
         for item in names_nbytes:
-            buffers[item.first] = ak._nplikes.numpy.empty(item.second)
```

Comment:
@ianna Is this "fix" correct? It's my understanding that we want to allocate `item.second` bytes here, so I'm setting a dtype of `uint8`.